run-dlp/README.md

```diff
@@ -0,0 +1,22 @@
+# run-dlp
```

Comment:
Can we have a more descriptive title something like "Data Loss Prevention workflow"?